### ISSUE TITLE
CodeGlance fills idea.log

### ISSUE THREAD

**Issue by cnanney** [Vektah/CodeGlance#111]
I notice that over 95% of my log entries are: `WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here.`

Thousands of these lines cluttering up the log. Is something wrong with the plugin?

PhpStorm (build #PS-143.1184.87, 09 Dec 2015 00:00)
CodeGlance 1.4.4

**Comment by WillieMaddox** [comment]
Same issue for me in webstorm.  But only 10 or so lines of WARN.  Still annoying.

**Comment by rquadling** [comment]
Just to give some idea of how useless this entry is ...

2016-11-23 12:16:14,000 [  65979]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,000 [  65979]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,000 [  65979]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,000 [  65979]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,000 [  65979]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,000 [  65979]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,031 [  66010]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,032 [  66011]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,032 [  66011]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,032 [  66011]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,032 [  66011]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,032 [  66011]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,032 [  66011]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,032 [  66011]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,032 [  66011]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,032 [  66011]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,032 [  66011]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,033 [  66012]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,033 [  66012]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,033 [  66012]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,033 [  66012]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,034 [  66013]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,034 [  66013]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,034 [  66013]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,034 [  66013]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,034 [  66013]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,034 [  66013]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,034 [  66013]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,035 [  66014]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,036 [  66015]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,036 [  66015]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,036 [  66015]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,036 [  66015]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,036 [  66015]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,036 [  66015]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,037 [  66016]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,037 [  66016]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,037 [  66016]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,037 [  66016]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,037 [  66016]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,038 [  66017]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,039 [  66018]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,040 [  66019]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,041 [  66020]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,042 [  66021]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,042 [  66021]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,042 [  66021]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,042 [  66021]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,042 [  66021]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,042 [  66021]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,042 [  66021]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,042 [  66021]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,042 [  66021]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,042 [  66021]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,043 [  66022]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 
2016-11-23 12:16:14,044 [  66023]   WARN - codeglance.EditorPanelInjector - I07: Injection skipped. Looks like we have already injected something here. 

That was in under 1/10th of a second.
